src/app/ui/NewTaskForm.tsx

```diff
@@ -219,23 +236,59 @@ const NewTaskFormInputs = () => {
           onChange={(e) => store.dispatch(setCreateTaskFields({ targetField: 'description', value: e.target.value }))}
         />
       </Stack>
+      <Stack direction="row" columnGap={2} m="16px 0px">
+        {attachments?.map((el, key) => {
+          return (
+            <Box key={key}>
```

Comment:
It's generally a good practice to not use index as a key, especially when indexes can change when attachments are deleted.

Please use attachment's id instead